app/scripts/models/auth_brokers/first-run.js

```diff
@@ -22,10 +22,6 @@ define([
       VERIFICATION_COMPLETE: 'verification_complete'
     },
 
-    haltAfterResetPasswordConfirmationPoll: false,
```

Comment:
This is one of the big reasons for this refactor, my intuition tells me all brokers should by default complete all flows as if the user was browsing directly to the sight. If a broker needs to override that behavior, it can. The newly extracted fx-desktop-v1 broker does just that.